### ISSUE TITLE
[Accessibility]: Incorrect focus on clicking Find References

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36568]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

On performing a screen reader click on the Find References button in the code intel popover, the screen reader either doesn't announce anything, or it announced stuff like:

> Large result set - only showing the first 500 results.
>
> Try out our new reference panel

(The latter is present if one is using the old ref panel)

However, the focus is not on the ref panel.

### Expected behavior

The screen reader should shift focus to the ref panel (and maybe the References section in the ref panel?) and announce some information about the context.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Closing as duplicate of https://github.com/sourcegraph/sourcegraph/issues/35924.

Solution here is to move the focus to the new panel when it is opened
